### DISCUSSION TITLE
Easier electrification of rails

### DISCUSSION THREAD

**Discussion by krnhns** [voxeltycoon/issues#886] [General] [answered]
Currently electrifying rails involves demolishing existing rails and laying new ones. Can the existing rails just be upgraded to avoid relaying the whole network?

**Comment by DanielBarreleiro** [comment, marked as answer]
There's a button just to electrify placed rails, no need to rebuild.
![image](https://user-images.githubusercontent.com/47695358/129477754-5f1a3c8e-71c4-4727-b1ba-616a5e79f2b1.png)

**Comment by krnhns** [comment]
Ok great I didn’t see this thanks for pointing it out
